src/governance.cpp

```diff
@@ -172,7 +175,10 @@ void CGovernanceManager::ProcessMessage(CNode* pfrom, std::string& strCommand, C
     else if (strCommand == NetMsgType::MNGOVERNANCEOBJECTVOTE)
     {
         // Ignore such messages until masternode list is synced
-        if(!masternodeSync.IsMasternodeListSynced()) return;
+        if(!masternodeSync.IsMasternodeListSynced()) {
+            LogPrintf("CGovernanceManager::ProcessMessage MNGOVERNANCEOBJECTVOTE -- masternode list not synced\n");
```

Comment:
This can easily be the case if at least one of our peers is in governance syncing mode (which means this node relays all received votes to us too). This multiplies with number of such peers but can quickly kill i/o even with one peer (given the fact that there will be 1000s of votes).